docs/userGuide/components/question.md

```diff
@@ -0,0 +1,183 @@
+
+<tip-box border-left-color="#00B0F0">
+  <i style="font-style: normal; font-weight: bold; color: dimgray">Example</i><br>
+  <question>
+    This question with an omitted hint.
+    <div slot="answer">42 <md>:star:</md></div>
+  </question>
+</tip-box>
+
+<tip-box border-left-color="#00B0F0">
+  <i style="font-style: normal; font-weight: bold; color: dimgray">Example</i><br>
+  <question>
+    This question with an omitted answer.
+    <div slot="hint">The number of exponent bits in a 64-bit Floating Point. <md>:computer:</md></div>
+  </question>
+</tip-box>
+
+<tip-box border-left-color="#00B0F0">
+  <i style="font-style: normal; font-weight: bold; color: dimgray">Example</i><br>
+  <question>
+    This question only has the question body. 
+  </question>
+</tip-box>
+
+<tip-box border-left-color="black">
+<i style="font-style: normal; font-weight: bold; color: dimgray">Markup</i>
+
+```html
+<question>
+  This question with an omitted hint.
+  <div slot="answer">42 <md>:star:</md></div>
+</question>
+
+<question>
+  This question with an omitted answer.
+  <div slot="hint">The number of exponent bits in a 64-bit Floating Point. <md>:computer:</md></div>
+</question>
+
+<question>
+  This question only has the question body. 
+</question>
+```
+</tip-box>
+<br>
+
+#### Use the minimized Panel component to create and group questions inline. 
+
+<tip-box border-left-color="#00B0F0">
+  <i style="font-style: normal; font-weight: bold; color: dimgray">Example</i><br>
+  <panel header="Q1 :crown:" minimized>
+    <question has-input>
+      Who is the first king of Thailand?
+    </question>
+  </panel>
+  <panel header="Q2 :pizza:" minimized>
+    <question has-input>
+      Which country did the Hawaiian pizza originate from? 
+      <div slot="hint">
+        Not Italy or Haiwaii <md>:smirk:</md>
+      </div>
+    </question>
+  </panel>
+</tip-box>
+
+<tip-box border-left-color="black">
+<i style="font-style: normal; font-weight: bold; color: dimgray">Markup</i>
+
+```html
+<panel header="Q1 :crown:" minimized>
+  <question has-input>
+    Who is the first king of Thailand?
+  </question>
+</panel>
+<panel header="Q2 :pizza:" minimized>
+  <question has-input>
+    Which country did the Hawaiian pizza originate from? 
+    <div slot="hint">
+      Not Italy or Haiwaii <md>:smirk:</md>
+    </div>
+  </question>
+</panel>
+```
+</tip-box>
+<br>
+
+#### Questions can also be loaded in from another file using Panel's `src` attribute.
+
+<tip-box border-left-color="#00B0F0">
+  <i style="font-style: normal; font-weight: bold; color: dimgray">Example</i><br>
+  <panel header="Questions loaded in from `quiz.md`" src="dynamic/quiz.md"></panel>
+</tip-box>
+
+<tip-box border-left-color="black">
+<i style="font-style: normal; font-weight: bold; color: dimgray">Markup</i>
+
+```html
+<panel header="Questions loaded in from `quiz.md`" src="dynamic/quiz.md"></panel>
+```
+</tip-box>
+<br>
```

Comment:
Might be a good time to add a `Question Options` table. :smirk:

Reply:
For one attribute? 😕

Reply:
For sake of completeness, probably yes. Maybe also list the names of the slots? The point of the table would be to give an easy list of attribute/slots reference without having to scout through the descriptions above.